contracts/mock/base/EtherCollateralLoansMock.sol

```diff
@@ -46,13 +46,17 @@ contract EtherCollateralLoansMock is EtherCollateralLoans {
         });
     }
 
-    function createEscrow(address borrower, uint256 loanID) public {
-        settings.getEscrowFactory().createEscrow(borrower, loanID);
+    function createEscrow(address borrower, uint256 loanID) public returns (address) {
```

Comment:
Why we are using this function?

I think
if it is to test the EscrowFactory.createEscrow function, we should use the EscrowFactory contract.
If it is to test takeOutLoan, it should use that function.

I already created an internal _createEscrow function (in LoansBase.sol -it is invoked in the takeOutLoan function-), and it can be mocked in the Loans mock contracts (see https://github.com/teller-protocol/teller-protocol-v1/blob/feature/escrow-contract-scaffolding/contracts/base/LoansBase.sol#L694)
Also, that function is already tested.

Reply:
https://github.com/teller-protocol/teller-protocol-v1/pull/137#discussion_r475918479